app/controllers/teams_controller.rb

```diff
@@ -23,6 +23,11 @@ def individuals
     end
   end
 
+  def plans
+    @user_group = UserGroup.find(params[:id])
```

Comment:
instead querying for user group in each action it may be done in before_action :find_user_group defined below as private ... or we may apply bigger refactoring and copy and simplify a bit ResourceController concern from eRostering added by Rob. It cuts the code in controller a lot but that should be a new story ,cuz it's a big refactoring https://github.com/sardjv/rostering/blob/master/app/controllers/concerns/resource_controller_concerns.rb